Making it so pandoc respects the dimensions of images in gdocs.

yelidrissi opened the pull request:
Lacking feature in the gdocs-export program that has been requested.

Depends on this previous pull request: https://github.com/dergachev/gdocs-export/pull/14

yelidrissi committed:
Update pandoc from version 1.12.2.1 to version 2.2.3.2 .

A lot of features are missing from the older version of pandoc that's used in the project, hence the motivation for the upgrade.

yelidrissi committed:
Making it so pandoc respects the dimensions of images in gdocs.

Lacking feature in the gdocs-export program that has been requested.

fbarbanson commented:
Approved.

dergachev commented on lib/include/preprocess.rb:
```diff
@@ -181,4 +182,14 @@ def validate_colspan
+  def fixup_image_attributes
+    doc.css("img").each do |img|
+      style = img.attr('style')
+      %w[height width].each do |att|
```
@yelidrissi please unwind this loop

yelidrissi committed:
Gem dependencies update.

yelidrissi committed:
Adding methods to PandocPreprocess that enable appropriate sizing of tables for Pandoc.

yelidrissi committed:
Update Ubuntu from trusty (2014 LTS) to bionic (2018 LTS)

 The original purpose of the update was to have a sufficiently modern version of ruby in the default package repositories, but it's also useful for multiple                              other reasons.
